content/posts/newsletter-december-2024.md

```diff
@@ -0,0 +1,32 @@
+title: Read the Docs Newsletter - December 2024
+date: December 11, 2024
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community
+authors: Eric Holscher
+status: published
+image: /images/december-2024.jpg
+image_credit: Photo by <a href="https://unsplash.com/@thebeardbe?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Filip Bunkens</a> on <a href="https://unsplash.com/photos/road-covered-by-snow-near-vehicle-traveling-at-daytime-R5SrmZPoO40?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+## News and updates
+
+* **Our new dashboard is now the default for all users.** This has been a long time coming, and we are excited to have have everyone migrated over. The old dashboard will be [removed on Mar 11th](https://about.readthedocs.com/blog/2024/11/rollout-of-our-new-dashboard/).
+* **[Link Previews](https://docs.readthedocs.io/en/stable/link-previews.html)** are now available in Addons. This feature allows you to see a preview of the linked page when you hover over a link in your documentation.
```

Comment:
I'm not sure how to write it, but it could be good to mention that this is tool agnostic and it works for other frameworks that are not Sphinx.

```suggestion
* **[Link Previews](https://docs.readthedocs.io/en/stable/link-previews.html)** are now available in Addons. This feature allows you to see a preview of the linked page when you hover over a link in your documentation. This is a re-implementation of our [Sphinx extension](https://sphinx-hoverxref.readthedocs.io/) that works for any documentation tool.
```